project-resources/community-health-files-guide.md

```diff
@@ -0,0 +1,61 @@
+# Guide to Community Health Files for OpenJS Foundation Projects
+
+Community health files are documents related to project collaboration. Most commonly, these include:
+
+- `CODE_OF_CONDUCT.md`
+- `CONTRIBUTING.md`
+- `GOVERNANCE.md`
+- `SECURITY.md`
+
+> [!NOTE]
+> See the [full list](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#supported-file-types) of supported default community health files
+
+## Default Community Health Files
```

Comment:
Can we reframe this whole section as: "here's a GitHub-specific thing that is helpful if you have multiple projects:"

Reply:
- It's not already framed that way?
- It's also still useful for orgs that only have one project